src/pyuvdata/utils/bls.py

```diff
@@ -388,18 +388,32 @@ def parse_ants(uv, ant_str, *, print_toggle=False, x_orientation=None):
 
 
 def _extract_bls_pol(
-    *, bls, polarizations, baseline_array, ant_1_array, ant_2_array, nants_telescope
+    *,
+    bls,
+    polarizations,
+    baseline_array,
+    ant_1_array,
+    ant_2_array,
+    nants_telescope,
+    invert=False,
+    strict=True,
```

Comment:
Can we add a docstring to this function? I know it's not required because it's a private function, but it's sufficiently complicated that I think a docstring will help future developers (including us).